tests/console/command_not_found.pm

```diff
@@ -16,6 +16,9 @@ use strict;
 sub run() {
     # permissions don't matter
 
+    # ensure command-not-found is installed (it is not part of minimal text mode installs for example)
```

Comment:
I think this is weakening the test. If you know of scenarios where it's *not meant* to be installed, install it there. But if it's supposed to be installed in a default install, then test that too.